include/data_structures/message.hpp

```diff
@@ -10,19 +10,18 @@ using namespace std;
 /*******************************************/
 /**************** Message_t ****************/
 /*******************************************/
-struct Message_t{
-    Message_t(){}
-    Message_t(float i_value)
-    :value(i_value){}
-    float   value;
-    void clear(){
-        Message_t(); 
-    }
-};
+    struct Message_t{
```

Comment:
Structure name should not start with a capital letter. So 'Message_t' should be changed to 'message_t'

Reply:
Its is a good suggestion, that structure name should not start by capital letter. But, no changes has been requested by professor in structure name. 
Even if I tried to do changes in structure name its breaking my code. I think it will be good if I we leave it without changing. Still please let me know what are your thoughts on that?

Reply:
Changes to the structure name has been done. Structure name changed from 'Message_t' to 'message_t'.